src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -1518,58 +1482,51 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
     }
   }
 
-  function getPriorityContext(
+  function getExpirationTime(
```

Comment:
This needs a better name. I was really confused what this does.

`getExpirationTimeForUpdatesOn(...)`?

I don't know...

Reply:
Agree it's confusing. Working on a PR that will remove this function entirely, so I'll leave it for now.

Reply:
Which PR?

Reply:
@blling https://github.com/facebook/react/pull/10715